compose.yaml

```diff
@@ -117,14 +112,13 @@ services:
     environment:
       - "RAILS_ENV=production"
       - "GEM_HOME=/gems"
-    group_add:
-      - ${OPENC3_LOCAL_MODE_GROUP_ID}
```

Comment:
By default, the docker container process runs with the supplementary groups looked up for the specified user. So is that why we can remove this?

Reply:
The container now runs as the user and the users default group.  So adding additional groups isn't necessary.